composites/Plugin/SnippetEditor/components/SnippetEditorFields.js

```diff
@@ -356,6 +470,7 @@ SnippetEditorFields.defaultProps = {
 		actual: 0,
 		score: 0,
 	},
+	mobileWidth: 356,
```

Comment:
Yes, this is meant to be able to adjust the magic number. But as you correctly pointed out. This should really be measured on the fly because we don't know how wide the label will be. Will create an issue to tackle this.